redhat-appstudio/service-provider-integration-operator
# Add a flag to disable HTTP/2 in the webhooks and kube-rbac-proxy

### What does this PR do?
`$TITLE`

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/SVPI-659

### How to test this PR?
See https://github.com/redhat-appstudio/remote-secret/pull/193

Test images: 
`quay.io/lkrejci/service-provider-integration-operator:svpi-659`
`quay.io/lkrejci/service-provider-integration-oauth:svpi-659`
